### PR TITLE
replace CR/LF to <br />

### PR DESCRIPTION
**Pull request by ryuzee**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -5,4 +5,8 @@ def js_prefix
     prefix = 'js' + Digest::MD5.hexdigest(SecureRandom.hex + Time.now.strftime('%Y%m%d%H%M%S'))
     prefix
   end
+
+  def nl2br(str)
```
Comment (ryuzee) [inline app/helpers/application_helper.rb]: __[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

File: app/views/slides/show.html.erb [hunk 2]
```diff
@@ -82,7 +82,7 @@ if @slide.tags.count > 0
    <div id="slide_div_box" style="padding:20px;">
     <div class="slide_detail_row">
         <span class="h3"><%= strip_tags @slide.name %></span>
-      <p><%= strip_tags @slide.description %></p>
+      <p><%= raw nl2br(sanitize(@slide.description)) %></p>
```
Comment (ryuzee) [inline app/views/slides/show.html.erb]: __[brakeman]__
* Unescaped model attribute [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting)

**Comment by satoryu** [comment]
Why don't you use [`simple_format`](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format)?

**Comment by ryuzee** [comment]
Ah, http://qiita.com/t_oginogin/items/0164ad76b753fdff9902

**Pull request merged**